tests/Makefile

```diff
@@ -4,6 +4,7 @@ export CFLAGS+=-g -O0 -Wall -D_GNU_SOURCE
 DISTRO = $(shell ./os_detect)
 
 SUBDIRS := \
+	exclude_filter \
```

Comment:
It tends to be "<noun>\_<verb>", so "filter_exclude" is _more correcter_ than "exclude_filter".